pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -89,10 +91,12 @@ def distribution_new_customer_spend(
     def expected_new_customer_spend(self) -> xarray.DataArray:
         """Expected transaction value for a new customer"""
 
-        assert self.idata is not None, "Model must be fitted first"
-        p_mean = self.idata.posterior["p"]
-        q_mean = self.idata.posterior["q"]
-        v_mean = self.idata.posterior["v"]
+        if (posterior := self.fit_result) is None:
```

Comment:
Same here

Reply:
Ahh got it! Fixed in https://github.com/pymc-labs/pymc-marketing/pull/637/commits/7e0689358e5f268dd6a8db50d1fad0ba025a9954